Automattic/wp-calypso
# composite-checkout: Add special formatting for bundled domain products

#### Changes proposed in this Pull Request

This modifies how line items are displayed in the review order step in composite checkout so that, if the line item is a bundled domain, it gets displayed with its original price crossed-out, its domain name, and a special "Free!" caption. (In addition, all domain products display their domain name.)

NOTE: the "Free" text was copied from `CartItem` and is what is used in the existing checkout. We might want to use different text.

Requires D40038-code

![Screen Shot 2020-03-06 at 7 39 20 PM](https://user-images.githubusercontent.com/2036909/76132984-cd9c2a00-5fe3-11ea-8ad7-95de9fb7cbdf.png)

Fixes #38978 

#### Testing instructions

- Apply D40038-code and sandbox the store.
- Visit composite checkout with a domain and a plan (you'll need to use the `composite-checkout-wpcom` flag to bypass the domain guard).
- Verify that the domain displays nicely as above and that the other products display normally.
